packages/flutter_tools/templates/app/.gitignore.tmpl

```diff
@@ -25,7 +25,8 @@
 .packages
 .pub-cache/
 .pub/
-/build/
+**/build/
+pubspec.lock
```

Comment:
Wouldn't some teams want pubspec.lock tracked in git?  To make sure that all members (and CI?) are getting exactly the same versions of packages?

Reply:
I figured they'd just pin their dependencies in pubspec.yaml instead, but that may be too restrictive.